BentoKit/BentoKit/Components/TitledDescription.swift

```diff
@@ -508,6 +508,35 @@ private extension Component.TitledDescription.View {
 }
 
 public extension Component.TitledDescription {
+    public final class ContentStyleSheet: ViewStyleSheet<BaseStackView> {
```

Comment:
This doesn't need to be `InteractiveViewStyleSheet`. In TD specifically, the highlighting gesture used is the one of the root view, and you can choose where the highlighting is applied to (root or content).

Reply:
My concern was from API point of view will it make sense to use the highlighting colour from root style sheet when highlighting target is `content`, or it would better to use highlighting colour from the content style sheet.